frontend/include/chpl/uast/prim-ops-list.h

```diff
@@ -401,3 +401,4 @@ PRIMITIVE_G(BREAKPOINT, "breakpoint")
 
 PRIMITIVE_G(CONST_ARG_HASH, "hash const arguments")
 PRIMITIVE_G(CHECK_CONST_ARG_HASH, "check hashes of const arguments")
+PRIMITIVE_R(TASK_INDEPENDENT_SVAR_CAPTURE, "task independent svar capture")
```

Comment:
Even though I know what you're doing, the name is not very self-documenting (I don't have a better alternative). Can you add a comment describing what this prim does?

Reply:
I'll add a comment to `compiler/AST/primitive.cpp` (since it looks like that's where we've commented on other prims).